src/run.ts

```diff
@@ -121,14 +121,8 @@ async function runLint(lintPath: string, patchPath: string): Promise<void> {
   const userArgs = core.getInput(`args`)
   const addedArgs: string[] = []
 
-  const userArgNames = new Set<string>()
-  userArgs
-    .split(/\s/)
-    .map((arg) => arg.split(`=`)[0])
-    .filter((arg) => arg.startsWith(`-`))
-    .forEach((arg) => {
-      userArgNames.add(arg.replace(`-`, ``))
-    })
+  const userArgNamesRegex = /(?<=(^|\s)-+\b)([^\s=]+)(?==)/ig
+  const userArgNames = new Set<string>(userArgs.match(userArgNamesRegex))
```

Comment:
```suggestion
  const userArgNames = new Set<string>()
  userArgs
    .split(/\s/)
    .map((arg) => arg.split(`=`)[0])
    .filter((arg) => arg.startsWith(`-`))
    .forEach((arg) => {
      userArgNames.add(arg.replace(/^-+/, ``))
    })
```

Reply:
I think that the real problem is here: the simple `replace` must be replaced by a trim prefix approach.

Reply:
I mean sure. Do it whichever which way, but if you are going to use a regex in the replace, you may as well skip all of the object iteration and just use a regex standalone. Fancy chained string operations are sweet and all, if you can find a performant reason to avoid regular expressions, but if you aren't, then the most performant option is a single step.

Either way is fine, if they both fix the bug. Your call.

You're gonna love the chained calls in my next PR... 😜

Reply:
Also, as a heads up: the original method (as well as your suggestion above) will hydrate the array with any space separated options, of which there are a few. My original PR won't. If we're shooting for stable behaviour.